### ISSUE TITLE
1.1/dev need help

### ISSUE THREAD

**Issue by ChristianGiupponi** [firesale/firesale#144]
Hi,
I never used git and I have no idea how to fork the 1.1dev to translate the files and send pull request, can someone please help me?

**Comment by chrisnharvey** [comment]
Could you try running the command `git checkout 1.1/develop`

**Comment by ChristianGiupponi** [comment]
No luck, I'm in the fork folder:

iMac-di-Christian:FireSale christiangiupponi$ git checkout 1.1/develop
error: pathspec '1.1/develop' did not match any file(s) known to git.
iMac-di-Christian:FireSale christiangiupponi$

**Comment by ChristianGiupponi** [comment]
I have found on google and this works, maybe...:

iMac-di-Christian:FireSale christiangiupponi$ git checkout origin/dev/1.1
HEAD is now at 69486ac... Added base of edit page
iMac-di-Christian:FireSale christiangiupponi$

EDIT:

ok I have donwloaded the branch, I still need to understand how to pull the local change to my online branch:

iMac-di-Christian:FireSale christiangiupponi$ git clone -b 1.1/develop git@github.com:firesale/FireSale.git 1.1
Cloning into 1.1...
remote: Counting objects: 3433, done.
remote: Compressing objects: 100% (883/883), done.
remote: Total 3433 (delta 2449), reused 3404 (delta 2430)
Receiving objects: 100% (3433/3433), 1.24 MiB | 619 KiB/s, done.
Resolving deltas: 100% (2449/2449), done.

EDIT2:

Done with:

git pull 1.1 and then sync :)
